include/eve/detail/skeleton.hpp

```diff
@@ -125,7 +129,9 @@ namespace eve::detail
     auto parts = kumi::make_tuple(slicer(ts)...);
 
     // Apply f on both side of the slices and re-combine
-    using wide_t = typename wide_result<Func, Ts...>::type;
+    using half_result_t = decltype(f(get<0>(slicer(ts))...));
+    using wide_t = typename half_result_t::template rescale<typename half_result_t::cardinal_type::combined_type>;
+
     return kumi::apply([&f](auto... m) { return wide_t { f(get<0>(m)...), f(get<1>(m)...)}; }, parts);
```

Comment:
kumi apply returns kumi tuple. I thought we  should retiurn wide. Am I wrong?

Reply:
apply takes a function f and a tupel of Ai and call f(Ai...), it returns w/e f is returning, here a wide